src/app/core/init.ts

```diff
@@ -0,0 +1,60 @@
+import StringValue from "./parameters/stringValue"
+import Component from "./component"
+import Connection from "./connection"
+import Feature from "./feature"
+
+type Point = [number, number]
+
+type Segment = number[]
+
+export type DeviceInterchangeV1 = {
+    name: StringValue
+    params: any
+    //layers: any[]
```

Comment:
'name' currently saved as StringValue, would it be better to save it as just a string?

Reply:
Yup, you should change it !